front/lib/solutions/start_worker.ts

```diff
@@ -0,0 +1,10 @@
+import { setupGlobalErrorHandler } from "@dust-tt/types";
```

Comment:
Let's use the same start_worker file and have a queue for solutions. It'll be less code an less infra work